### ISSUE TITLE
Org-alert expected behaviour

### ISSUE THREAD

**Issue by pseudomyne** [spegoraro/org-alert#21]
Hi. 
I'm struggling with notifications for org deadline or scheduled events. 
Basically, what I want is to be notified (dunst here) _at the time_ I scheduled / deadlined any agenda file entry. 

Is this what org-alert does ? 
It seems to me that it is not the case : `org-alert` just notifies _periodically_, but _not_ starting at the time specified in the entry. 
Am I right ? If so, what do you recommend me fot this basic need ?  `appt` ? [Org wild notifier](https://github.com/akhramov/org-wild-notifier.el) ?
I find it confusing that this need is not fullfilled in default org-agenda settings.

**Comment by ntBre** [comment]
This use case should now be handled after https://github.com/spegoraro/org-alert/commit/e9293b924b61de8fda42fd4bc5717f40e2db31d4, which added `org-alert-notify-cutoff` to let you set how long before an event the notification is sent. The default is 10 minutes before, but you should be able to set it to 0 to send the alert at the scheduled time.

**Comment by NotQuiteAwake** [comment]
Hi,

This may just be some basic misunderstanding, but is it expected that org-alert only sends out notifications a set time before the _end_ of a scheduled event? I have used the snippet in the README:

```elisp
(setq alert-default-style 'libnotify
      org-alert-interval 300
      org-alert-notify-cutoff 10
      org-alert-notify-after-event-cutoff 10)
```

Suppose it's now 17:00 and I have an event scheduled from 17:05 to 17:40. It seems I will get no notifications before 17:30. On the other hand, if I had scheduled an event at 17:05, with no end time, which to me means that I will start working on this at 17:05, I would start receiving notifications at 16:55.

If this is indeed the expected behaviour in the current code, I would love it if we could set up alerts at the beginning of a scheduled time window as well.

Thanks!
